docs/intents/spec.md

```diff
@@ -31,29 +37,61 @@ Intents shoulde be:
 All standard intent names are reserved. Applications may use their own intents ad hoc.
 However, there is a need for applications to ensure that their intents avoid collision. The recommended approach here is to use the app name as the noun. For example, the ‘myChart’ App may expose the ‘ViewChart’ intent and the ‘myChart.Foo’ proprietary intent.
 
+### Intents
+
+The FDC3 2.0 standard expanded the Intents that are available.  The list of available Intents are the following:
+
+### `View___`
+  * Expected context: Contact
+
+### `Start___`
+  * Expected context: Contact
+
+### `Create___`
+  * Expected characteristics: Create a new record, but fail if it already exists
+
+### `Update___`
+  * Expected characteristics: Update an existing record, but fail if it does not exist
+
+### `CreateOrUpdate___`
+  * Expected characteristics: todo
+
+### `Delete___`
+  * Expected characteristics: Delete an existing record, but fail if it does not exist
+
+### `Get___`
+  * Expected characteristics: Retrieve an existing record, but fail if it does not exist
+
+### `Share___`
+  * Expected characteristics: Share an existing record, but fail if it does not exist
+
+
+
+
+
 ## Initial Set of Standard Intents ##
```

Comment:
I'm not sure we need this anymore and could replace with a comment that standardized intents are defined in the following pages, otherwise just cut it down to a bulleted list with each entry having a link to the relevant page...

Reply:
Hi @kriswest is there a way to  have a directory of the Ref/ folde? As this will allow future intents to appear as and when they are added?

As currently I need to include a hyperlink for each definition.

Reply:
We've got the sidebar nav, but thats all. Hence, we either remove the list and just mention that the individual types are on subsequent pages, or manually add a link for each... 

It would be fine (and probably easier) to do this all in a subsequent PR when we've got the full set of intent and context PRs lined up.